src/stores/global.ts

```diff
@@ -5,25 +5,25 @@ export const useGlobalStore = defineStore('global', {
     isDarkTheme: false,
     coursesLists: [],
     paginatedCoursesLists: [],
-    course: {},
```

Comment:
Suggestion: I think this comma makes sense, because the background makes it easier to extend the functionality further